craft_providers/multipass/multipass_instance.py

```diff
@@ -189,9 +195,11 @@ def execute_run(
         :raises subprocess.CalledProcessError: if command fails and check is
             True.
         """
+        cwd = kwargs.pop("cwd", None)
```

Comment:
Why `cwd` can't be expressed in the method's signature?

Reply:
Good point, simplifies some things with a bit of extra work. done! :D